extensions/wikia/AdEngine/js/run/desktop.run.js

```diff
@@ -142,6 +143,10 @@ require([
 		if (exitstitial) {
 			exitstitial.init();
 		}
+
+		if (floatingMedrec) {
+			win.wgAfterContentAndJS.push(floatingMedrec.init);
```

Comment:
I'd rather change this line to `floatingMedrec.init()` to keep consistent with the rest since it's being executed after DOM loads anyway (lines 152-156). And I think it's important to init after DOM loads because having content and JS isn't good enough since there were issues where users' JS were breaking the functionality.